### DISCUSSION TITLE
Use of defaultRevision field for injection

### DISCUSSION THREAD

**Discussion by dgn** [istio/istio#50692] [General]
I noticed we added a [`defaultRevision` field](https://github.com/istio/istio/blob/e48725f676233fcbc1af67c2089c5ce5ed35feac/manifests/charts/base/values.yaml#L38) way back in 2021 and it seems to make sure to point the ValidatingWebhook to that revision, which I think is cool when doing canary upgrades. It doesn't however affect injection - I would have expected that if I set the `defaultRevision` to a revision, it will allow me to use the istio-injection label again (instead of having to use `istio.io/rev`), but that doesn't seem to be the case. Is there interest in that? If so, I'll implement it

**Comment by howardjohn** [comment]
I had thought for sure this was how it works. Is that not the case?

I recall we were using it for this purpose in a former deployment i worked on, but maybe we had some customization..
